eclass-writing/text.xml

```diff
@@ -187,7 +187,7 @@ to properly break lines when generating eclass manpages.
 
 <p>
 An eclass is a <c>bash</c> script which is sourced within the ebuild environment.
-Files should be a simple text file with a <c>.eclass</c> extension. Allowed
+Files should be a simple text file with an <c>.eclass</c> extension. Allowed
```

Comment:
This can be pronounced as "a dot eclass extension", in which case "a" is correct.

Reply:
I see. reverted